test/dom/index.js

```diff
@@ -14,6 +30,44 @@ describe('dom', () => {
     assert.equal(result.msTransform, result.transform, 'sets prefixed styles');
   });
 
+  it('media', () => {
+    const { mediaS, mediaM, mediaL } = media;
+
+    assert.equal(mediaS, '(min-width: 450px)', 'MediaS is 450px');
+    assert.equal(mediaM, '(min-width: 690px)', 'MediaM is 690px');
+    assert.equal(mediaL, '(min-width: 920px)', 'MediaL is 920px');
+  });
+
+  it('getMedia', () => {
+    const { mediaS, mediaM, mediaL } = media;
+
+    const node1 = {
+      media: '(min-width: 450px) (max-width: 800px)',
+      matchMedia(query) {
+        return { matches: this.media.includes(query) };
```

Comment:
you need to create a spy and check that it's being called properly, nothing more. check how spies work here https://sinonjs.org/releases/v8.1.1/spy-call/